package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -449,19 +450,27 @@ def write(self, selection, frame=None):
 class DumpReader(base.ReaderBase):
     """Reads the default `LAMMPS dump format`_
 
-    Expects trajectories produced by the default 'atom' style dump.
-
-    Will automatically convert positions from their scaled/fractional
-    representation to their real values.
+    Supports coordinates in the "unscaled" (x,y,z), "scaled" (xs,ys,zs),
+    "unwrapped" (xu,yu,zu) and "scaled unwrapped" (xsu,ysu,zsu) coordinate
+    conventions. If coordinates are given in the scaled coordinate convention
+    (xs,ys,zs), they will automatically be converted from their
+    scaled/fractional representation to their real values.
 
     .. versionadded:: 0.19.0
     """
     format = 'LAMMPSDUMP'
+    _conventions = ["guess", "unscaled", "scaled", "unwrapped",
+                    "scaled_unwrapped"]
 
-    def __init__(self, filename, **kwargs):
+    def __init__(self, filename, coordinate_convention="guess", **kwargs):
         super(DumpReader, self).__init__(filename, **kwargs)
 
         root, ext = os.path.splitext(self.filename)
+        if coordinate_convention in self._conventions:
+            self.coordinate_convention = coordinate_convention
+        else:
+            raise ValueError("coordinate convention incorrectly specified")
```

Comment:
Please do make sure all the things that codecov complain about get appropriate tests.

Reply:
Yeah more tests needed for sure.